Gruntfile.js

```diff
@@ -441,13 +441,7 @@ module.exports = (grunt) => {
   })
 
   // Useful helper task for `grunt test`.
-  grunt.registerTask('backend:launch', '[internal]', function () {
-    const done = this.async()
-    grunt.log.writeln('backend: launching...')
-    // Provides Babel support for the backend files.
-    require('@babel/register')
-    require(backendIndex).then(done).catch(done)
-  })
+  grunt.registerTask('backend:launch', '[internal]', ['backend:relaunch'])
```

Comment:
It seems odd that this would be needed instead of just renaming `'backend:relaunch'` to `'backend:launch'` and deleting this section.

Note to self: make sure to test this with:

- editing and saving backend files to verify server relaunches and the page refreshes afterward
- verify `grunt test` also still works (I guess it does as otherwise tests would fail)

Reply:
> editing and saving backend files to verify server relaunches and the page refreshes afterward

Yeah so the server relaunches, but the page does not refresh.